shared/src/vyxal/ListHelpers.scala

```diff
@@ -649,4 +649,20 @@ object ListHelpers:
         case a: VList => vectorisedMinimum(a, b)
         case a: VVal => MiscHelpers.dyadicMinimum(a, b)
     })
+
+  def uniquify(iter: VList): VList =
```

Comment:
Sorry, this should be

```scala
def uniquify(iter: VList)(using ctx: Context): VList =
```

For some reason I can't remember the uniquify function needs context.